guides/source/command_line.md

```diff
@@ -508,11 +508,11 @@ $ bin/rails destroy model Oops
 $ bin/rails about
 About your application's environment
 Rails version             7.2.0
-Ruby version              2.7.0 (x86_64-linux)
```

Comment:
We generally bump when dropping supported versions not the latest, else we would bump it regularly